.github/workflows/smoke-test.yaml

```
@@ -3,63 +3,85 @@
     branches:
       - dev
 
 jobs:
+  skip_if_only_docs:
+    runs-on: ubuntu-latest
+    outputs:
+      only_docs: ${{ steps.changes.outputs.only_docs }}
+      
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)